src/Chapter4.hs

```diff
@@ -488,7 +495,18 @@ Implement the 'Applicative' instance for our 'List' type.
   may also need to implement a few useful helper functions for our List
   type.
 -}
+appendLists :: List a -> List a -> List a
+appendLists Empty Empty = Empty
+appendLists Empty (Cons y ys) = Cons y (appendLists Empty ys)
```

Comment:
These two cases can be combined into a single one. Note, that when the first list is `Empty`, you can just return the second list :slightly_smiling_face: